src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-item {
+  margin-left: 20px;
+}
+
+.link {
+  display: flex;
+  position: relative;
+  align-items: center;
+  height: 60px;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: hsl(0, 0%, 0%);
+}
+
+.is-active {
+  position: relative;
+  color: hsl(193, 100%, 43%);
+}
+
+.link:hover {
+  color: hsl(193, 100%, 43%);
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  border-radius: 2px;
+  box-sizing: border-box;
```

Comment:
It is a good tone to set box-sizing for all elements(*) and don't change it without extra need